storage/src/mmr/benches/prove_many_elements.rs

```diff
@@ -1,5 +1,5 @@
 use commonware_cryptography::{Hasher, Sha256};
-use commonware_storage::mmr::mem::Mmr;
+use commonware_storage::mmr::{mem::Mmr, verification::VerifiableMmr};
```

Comment:
is there a non-verifiable MMR in verification?

Reply:
nit: I would call this `verification::Storage`

Reply:
Renamed "Provable"

Reply:
Renamed "Storage"